tests/it.xsemantics.dsl.tests/test-gen/fj_lambda_test/it/xsemantics/test/fj/lambda/FjTestsForLambdas.java

```diff
@@ -17,14 +27,7 @@
 import org.eclipse.xtext.xbase.lib.ListExtensions;
 
 /**
- * Copyright (c) 2013-2017 Lorenzo Bettini.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- * 
- * Contributors:
- *   Lorenzo Bettini - Initial contribution and API
+ * Uses rule invocations as boolean expressions
```

Comment:
You removed the Javadoc with @author tag... with @mor-n4 we decided to leave these ones... if it's only in test files, it's not a problem, but please don't remove them from standard files.